includes/sm-template-functions.php

```diff
@@ -634,8 +634,15 @@ function wpfc_sermon_excerpt_v2( $return = false, $args = array() ) {
 					</div>
 				<?php endif; ?>
 			</div>
-			<?php $sermon_description = get_post_meta( $post->ID, 'sermon_description', true ); ?>
-			<div class="wpfc-sermon-description"><?php echo wp_trim_words( $sermon_description, 30 ); ?></div>
+
+			<div class="wpfc-sermon-description">
+                <?php if ( has_excerpt( $post ) ) : ?>
+                    <?php echo get_the_excerpt( $post ); ?>
+                <?php else: ?>
+                    <?php echo wp_trim_words( get_post_meta( $post->ID, 'sermon_description', true ), 30 ); ?>
```

Comment:
Nicely done! Just make sure that it follows CS (tabs vs spaces)

Reply:
Fixed